by Nico Golde <nion@debian.org>
the freedesktop standard requires some additional attributes in the desktop files
which are added here to fix #533126, #531580
Index: openbox-3.4.11.1/data/gnome-wm-properties/openbox.desktop
===================================================================
--- openbox-3.4.11.1.orig/data/gnome-wm-properties/openbox.desktop	2010-04-23 16:47:53.000000000 +0200
+++ openbox-3.4.11.1/data/gnome-wm-properties/openbox.desktop	2010-04-23 16:48:08.000000000 +0200
@@ -2,10 +2,15 @@
 Type=Application
 Name=Openbox
 Exec=openbox
+Type=Application
+Categories=System;
 
 # name we put on the WM spec check window
 X-GNOME-WMName=Openbox
 
+X-GNOME-Provides=windowmanager
+NoDisplay=true
+
 # our config tool
 ConfigExec=obconf
 
